frontend/src/metabase/components/MetadataInfo/FieldFingerprintInfo/FieldFingerprintInfo.jsx

```diff
@@ -71,9 +71,18 @@ function NumberFingerprint({ className, field }) {
   }
 
   const { avg, min, max } = numberFingerprint;
-  const fixedAvg = formatNumber(Number.isInteger(avg) ? avg : avg.toFixed(2));
-  const fixedMin = formatNumber(Number.isInteger(min) ? min : min.toFixed(2));
-  const fixedMax = formatNumber(Number.isInteger(max) ? max : max.toFixed(2));
+
+  function roundNumber(num, digits = 2) {
+    if (num === null) {
+      return "";
```

Comment:
@daltojohnso not sure what would be good to display in this case? An empty string, `0` or something else?

Reply:
Uhm... just realized I am fixing this in the wrong place.
The scenario I was looking at was when ALL values are `null`, like the OP posted in his screenshot.

But if we have `[NULL, 0.25, NULL, 5, 3.11]`, then the **average** should be 2.79, **min** should be 0?, and **max** should be 5.

Reply:
If the value is `null` we should probably hide that part of the fingerprint entirely, I think. That means wrapping the associated `<th>...</th>` like so: `{fixedAvg ? <th>{...}</th> : undefined}` and doing the same for the associated `<td>...</td>`

We should also make sure that at least one of `fixedAvg`, `fixedMin`, or `fixedMax` is defined. Otherwise, make the component return `null`.